src/main/java/sx/blah/discord/handle/impl/obj/Guild.java

```diff
@@ -847,6 +845,23 @@ public IEmoji getEmojiByName(String name) {
 	}
 
 	@Override
+	public IEmoji createEmoji(String name, Image image, IRole... roles) {
+		if (name.length() < 2 || name.length() > 32 || DiscordUtils.EMOJI_NAME_PATTERN.matcher(name).find())
+			throw new DiscordException("Emoji name must be 2-32 alphanumeric characters and underscores.");
```

Comment:
Weird wording. I think a better message would be "Emoji name must be between 2-32 characters and consist only of alphanumeric characters and underscores."

Reply:
I was copying the message @danthonywalker asked me to copy. Shall I still change this one?

Reply:
The current message is ambiguous. It could imply that you are allowed between 2 and 32 alphanumerics but any number of underscores.